_seminars/fall2016/index.md

```diff
@@ -2,4 +2,77 @@
 title: "Fall 2016"
 year: 2016
 semester: fall
----
\ No newline at end of file
+
+papers:
+  - title: Modeling Facial Expressions in 3D Avatars from 2D Images
+    author: Emma Sax
+    pdf: Sax.pdf
+    slides: Saxslides.pdf
+    time: 2016-11-12 1:00 PM
+
+  - title: Computer Virus Enhancement
+    author: John P. Lynch
+    pdf: Lynch.pdf
+    slides: Lynchslides.pdf
+    time: 2016-11-12 1:30 PM
+
+  - title: Supporting Literary Analysis Using Automatically Generated Social Network Graphs
+    author: Jocelyn M. Bayer
+    pdf: Bayer.pdf
+    slides: Bayerslides.pdf
+    time: 2016-11-12 2:00 PM
+
+  - title: Adding functional stye pattern matching features to object-oriented languages
+    author: Joseph Thelen
+    pdf: Thelen.pdf
+    slides: Thelenslides.pdf
+    time: 2016-11-12 2:45 PM
+
+  - title: Abstracting Natural Language Queries into SQL
+    author: Thomas Hagen
+    pdf: Hagen.pdf
+    slides: Hagenslides.pdf
+    time: 2016-11-12 3:15 PM
+
+  - title: Modern Approaches to the Rich Vehicle Routing Problem
+    author: Henry F.R. Fellows
+    pdf: Fellows.pdf
+    time: 2016-11-19 1:00 PM
+
+  - title: Abstractive Automatic Text Summarization
+    author: Isaac Koak
+    pdf: Koak.pdf
+    slides: Koakslides.pdf
+    time: 2016-11-19 1:30 PM
+
+  - title: Improving security of the Advanced Encryption Standard
+    author: Mark M. Lehet
+    pdf: Lehet.pdf
+    slides: Lehetslides.pdf
+    time: 2016-11-19 2:00 PM
+
+  - title: SQL and NoSQL: Comparing Modern Databases
```

Comment:
This will make the YAML invalid, if an element contains any YAML special characters like `:`, please wrap in quotes.